@kiva/kv-shop/src/tests/basket.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { getCookieValue } from '../basket';
```

Comment:
Doesn't need to change, but often the following happens in jest tests.

- A `describe` for each method, for example a second `describe('getCookieValue')`
- `it` strings typically start with "should", for example `it('should encode token')` for readability in test output